mne/viz/evoked.py

```diff
@@ -311,10 +311,18 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                         for i in bad_ch_idx:
                             if i in idx:
                                 colors[idx.index(i)] = 'r'
-                    for ch_idx in range(len(D)):
+
+                    # find the channels with the least activity,
+                    # and map them in front of the more active ones
+                    auc = np.trapz((np.abs(D - D.mean(1).reshape(-1, 1))))
```

Comment:
![screen shot 2016-04-30 at 22 48 03](https://cloud.githubusercontent.com/assets/4321826/14938497/a5eb873e-0f25-11e6-8c4c-b0f919d36b63.png)

This is with STD